src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -83,7 +108,7 @@ FULL_PATH_BUILD_DIR=`pwd`
 
 : ${CYPRESS_CACHE_FOLDER:="${FULL_PATH_BUILD_DIR}/src/pybind/mgr/dashboard/cypress"}
 
-export CYPRESS_BASE_URL CYPRESS_CACHE_FOLDER CYPRESS_LOGIN_USER CYPRESS_LOGIN_PWD NO_COLOR
+export CYPRESS_BASE_URL CYPRESS_CACHE_FOLDER CYPRESS_LOGIN_USER CYPRESS_LOGIN_PWD NO_COLOR CYPRESS_CEPH2_URL
```

Comment:
Its easier if I put a prefix of `CYPRESS_` for the env variable we want to use. Their is no strict guideline from [cypress I guess for doing like this](https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_)